src/main/webapp/app/entities/event-activity/event-activity.tsx

```diff
@@ -53,98 +46,104 @@ export class EventActivity extends React.Component<IEventActivityProps, IEventAc
   handlePagination = activePage => this.setState({ activePage }, () => this.sortEntities());
 
   getEntities = () => {
+    const { eventId } = this.props.match.params;
     const { activePage, itemsPerPage, sort, order } = this.state;
-    this.props.getEntities(activePage - 1, itemsPerPage, `${sort},${order}`);
+    this.props.getEventActivitiesByEventId(Number.parseInt(eventId, 10), activePage - 1, itemsPerPage, `${sort},${order}`);
+  };
+
+  showCardAction = (eventActivityId: number) => {
+    this.props.setSelectedEventActivityId(eventActivityId);
+    this.props.setShowActionOptions(true);
+  };
+
+  toggleShowOptions = () => {
+    this.props.setShowActionOptions(!this.props.showActionOptions);
   };
 
   render() {
-    const { eventActivityList, match, totalItems } = this.props;
+    const { eventActivityList, match, totalItems, selectedEventActivityId } = this.props;
     return (
       <div>
         <h2 id="event-activity-heading">
           <Translate contentKey="clubmanagementApp.eventActivity.home.title">Event Activities</Translate>
-          <Link to={`${match.url}/new`} className="btn btn-primary float-right jh-create-entity" id="jh-create-entity">
+        </h2>
+        <CustomTab tabList={eventTabList} currentTab="Activities" />
```

Comment:
Shouldn't that the eventTabList have an argument of `eventId` due to the updated `tab.constants.ts`?

Reply:
resolved in 948ee18a02eeba1d00248750fb41673c50daa860